Doc/whatsnew/3.10.rst

```diff
@@ -82,9 +82,28 @@ New Features
 * :pep:`618`: The :func:`zip` function now has an optional ``strict`` flag, used
   to require that all the iterables have an equal length.
 
-* :pep:`604`: Builtin Union type to allow writing X | Y in place of
-  ``typing.Union[X, Y]``.
-  (Contributed by Maggie Moss and Philippe Prados in :issue:`41428`.)
+PEP604: New Type Operator
+-------------------------
+A new type union operator was introduced which enables the syntax ``X | Y``.
+This provides a cleaner way of expressing 'either type X or type Y' instead of
+using :data:`typing.Union`.
```

Comment:
```suggestion
This provides a cleaner way of expressing 'either type X or type Y' instead of
using :data:`typing.Union`, especially in type hints (annotations).
```